core/backend/src/BackendRequestContext.ts

```diff
@@ -21,7 +21,7 @@ export class AuthorizedBackendRequestContext extends AuthorizedClientRequestCont
    * Create a new context for agent applications or long running backend operations to pass to various services
    * @see [[AuthorizedBackendRequestContext.create]] to create the request based on the authorization information supplied to IModelHost.
    */
-  public constructor(accessToken: AccessToken, activityId: string = Guid.createValue()) {
+  public constructor(accessToken?: AccessToken, activityId: string = Guid.createValue()) {

```

Comment:
@calebmshafer the whole concept of `BackendRequestContext` and `AuthorizedBackendRequestContext` don't make any sense. `RequestContext` and activityId refer to an RPC request - which can only originate from a frontend. Shouldn't this just be accessToken?

Reply:
@kabentley and I talked and he's going to start flattening the hierarchy of request context and removing the AccessToken from it. Then any method that needs both will take both as args.  However, that'll be a separate PR after this one.